opencti-platform/opencti-front/tests_e2e/enableReferences/addReportObservable.spec.ts

```diff
@@ -0,0 +1,203 @@
+import { expect, test } from '../fixtures/baseFixtures';
+import ReportPage from '../model/report.pageModel';
+import ContainerObservablesPage from '../model/containerObservables.pageModel';
+import ReportDetailsPage from '../model/reportDetails.pageModel';
+import ReportFormPage from '../model/reportForm.pageModel';
+import LoginPage from '../model/login.pageModel';
+import DashboardPage from '../model/dashboard.pageModel';
+import CommitMessagePage from '../model/commitMessage.pageModel';
+import ContainerAddObservablesPage from '../model/containerAddObservables.pageModel';
+import RolesSettingsPage from '../model/rolesSettings.pageModel';
+import RoleFormPage from '../model/roleForm.pageModel';
+import RolePage from '../model/role.pageModel';
+import GroupsSettingsPage from '../model/groupsSettings.pageModel';
+import GroupPage from '../model/group.pageModel';
+import GroupFormPage from '../model/groupForm.pageModel';
+import UsersSettingsPage from '../model/usersSettings.pageModel';
+import UserPage from '../model/user.pageModel';
+import UserFormPage from '../model/userForm.pageModel';
+
+const noBypassUserAuthFile = 'tests_e2e/.setup/.auth/no-bypass-ref-user.json';
+const noBypassUserLogin = 'noBypassReferences@user.test';
+const noBypassUserPassword = 'qwerty123';
+const noBypassUserName = 'NoBypassReferencesUser';
+const noBypassRoleName = 'NoBypassReferencesRole';
+const noBypassGroupName = 'NoBypassReferencesTestGroup';
+
+test.describe('Create user with no references bypass capabilities', () => {
+  test('Create basic user role', async ({ page }) => {
+    const rolesSettingsPage = new RolesSettingsPage(page);
+    const rolePage = new RolePage(page);
+    const roleFormPage = new RoleFormPage(page);
+
+    await page.goto('/dashboard/settings/accesses/roles');
+    await expect(rolesSettingsPage.getSettingsPage()).toBeVisible();
+    await rolesSettingsPage.getAddRoleButton().click();
+    await roleFormPage.fillNameInput(noBypassRoleName);
+    await roleFormPage.getCreateButton().click();
+    await expect(rolesSettingsPage.getRoleInList(noBypassRoleName)).toBeVisible();
+    await rolesSettingsPage.getRoleInList(noBypassRoleName).click();
+    await rolePage.getEditButton().click();
+    await roleFormPage.getCapabilitiesTab().click();
+    await roleFormPage.getAccessKnowledgeCheckbox().click();
+    await expect(roleFormPage.getAccessKnowledgeCheckbox()).toBeChecked();
+    await roleFormPage.getCreateUpdateKnowledgeCheckbox().click();
+    await expect(roleFormPage.getCreateUpdateKnowledgeCheckbox()).toBeChecked();
+    await roleFormPage.getAccessAdministrationCheckbox().click();
+    await expect(roleFormPage.getAccessAdministrationCheckbox()).toBeChecked();
+  });
```

Comment:
Random thinking: I guess we could have some way to reuse that piece of test code that creates user/roles and groups I don't mean one groups for all test, but have a methode like createRole(roleName, listOfCapacities) ...etc

Reply:
That's a good point, we already talked about that with Souad and Jpk actually. We are planning to discuss it more during the e2e workshop